pallets/projects/src/mock.rs

```diff
@@ -176,7 +176,7 @@ impl pallet_timestamp::Config for Test {
 
 parameter_types! {
     pub const TwoWeekBlockUnit: u32 = 100800u32;
-    pub const ProposalsPalletId: PalletId = PalletId(*b"imbgrant");
+    pub const ProjectsPalletId: PalletId = PalletId(*b"imbgrant");
```

Comment:
This looks wrong to me, maybe you can look if this value ProjectsPalletId has been changed at all.